adafruit_pca9685.py

```diff
@@ -82,7 +82,7 @@ def duty_cycle(self) -> int:
     @duty_cycle.setter
     def duty_cycle(self, value: int) -> None:
         if not 0 <= value <= 0xFFFF:
-            raise ValueError("Out of range")
+            raise ValueError(f"Out of range: value {value} not 0 <= value <= 0xFFFF")
```

Comment:
I like the change, but let's use decimal instead of hex since that's more beginner friendly.